onnx/helper.py

```diff
@@ -563,7 +563,10 @@ def make_attribute(
         attr.type = AttributeProto.INT
     # string
     elif bytes_or_false is not False:
-        assert isinstance(bytes_or_false, bytes)
+        if not isinstance(bytes_or_false, bytes):
+            raise ValueError(
+                "{} must be an instance of {}.".format(bytes_or_false, bytes)
+            )
```

Comment:
Thanks for this PR. looks like this error message is "??? must be an instance of <class 'bytes'>.". Is it better to post "value must be an instance of <class 'bytes'>."

Reply:
Thanks for the comment, I updated the message.